src/aquarium-optimized/Aquarium.cpp

```diff
@@ -875,79 +882,22 @@ void Aquarium::updateAndDrawFishes() {
                static_cast<float>(M_PI) * 2),
           ii);
 
-      model->updatePerInstanceUniforms(worldUniforms);
-      model->draw();
+      if (mDrawPerModel) {
```

Comment:
This is a branch in the hottest path of Aquarium, but I don't know if that still matters in 2020.